notebooks/official/workbench/inventory-prediction/inventory_prediction.ipynb

```diff
@@ -29,21 +29,24 @@
         "id": "565d260c8eda"
```

Comment:
<p>for links, use format:</p><p> </p><p>Lear more about [title](link).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='101'/>